src/code_f8f60_len_1560.c

```diff
@@ -263,12 +253,14 @@ ApiStatus GetDist2D(ScriptInstance* script, s32 isInitialCall) {
     Bytecode* ptrReadPos = script->ptrReadPos;
 
     Bytecode outVar = *ptrReadPos++;
-    set_float_variable(script, outVar, dist2D(
-        get_float_variable(script, *ptrReadPos++),
-        get_float_variable(script, *ptrReadPos++),
-        get_float_variable(script, *ptrReadPos++),
-        get_float_variable(script, *ptrReadPos++)
-    ));
+    set_float_variable(
+        script,
+        outVar,
+        dist2D(
+            get_float_variable(script, *ptrReadPos++),
+            get_float_variable(script, *ptrReadPos++),
+            get_float_variable(script, *ptrReadPos++),
+            get_float_variable(script, *ptrReadPos++)));
```

Comment:
Maybe worth extracting the distance to a variable above to make this more readable?

Reply:
Ideally clang-format would format multiline function calls like the following, but I'm not sure how to get this behaviour.
```diff
  foo(
    arg0,
    arg1,
-   arg2)
+   arg2
+ )
```

Reply:
Yeah...I think this is definitely a bummer but not a huge deal